tools/launchdev.sh

```diff
@@ -183,7 +183,9 @@ function st2clean(){
     mongo st2 --eval "db.dropDatabase();"
     # start with clean logs
     LOGDIR=$(dirname $0)/../logs
-    rm ${LOGDIR}/*
+    if [ "$(ls -A ${LOGDIR} | wc -l)" != "0" ]; then
```

Comment:
if [ -d $LOGDIR ] should test directory exists. Wondering why the ls -A | wc -l?